stream_alert_cli/terraform_generate.py

```diff
@@ -352,6 +352,32 @@ def generate_cloudtrail(cluster_name, cluster_dict, config):
     """
     modules = config['clusters'][cluster_name]['modules']
     cloudtrail_enabled = bool(modules['cloudtrail']['enabled'])
+    existing_trail_default = False
+    existing_trail = modules['cloudtrail'].get('existing_trail', existing_trail_default)
+    is_global_trail_default = True
```

Comment:
Correct, they're just booleans.  I defined them as variables to be really explicit about what the defaults are for these options.